app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/EventDetailFragment.java

```diff
@@ -159,7 +164,22 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             if (lecture != null && lecture.dateUTC > 0) {
                 DateFormat df = SimpleDateFormat
                         .getDateTimeInstance(SimpleDateFormat.SHORT, SimpleDateFormat.SHORT);
-                t.setText(df.format(new Date(lecture.dateUTC)) + " - " + room);
+
+                Drawable drawable_ACESS_TIME = getContext().getResources().getDrawable(R.drawable.ic_access_time);
+                Drawable drawable_ROOM = getContext().getResources().getDrawable(R.drawable.ic_room);
+
+                drawable_ACESS_TIME.setBounds(0, 0, (int)t.getTextSize(), (int)t.getTextSize());
+                drawable_ROOM.setBounds(0, 0, (int)t.getTextSize(), (int)t.getTextSize());
+
+                ImageSpan imageSpan_ACESS_TIME = new ImageSpan(drawable_ACESS_TIME, ImageSpan.ALIGN_BASELINE);
+                ImageSpan imageSpan_ROOM = new ImageSpan(drawable_ROOM, ImageSpan.ALIGN_BASELINE);
+
+                SpannableString spannableString = new SpannableString("  " + df.format(new Date(lecture.dateUTC)) + " -  " + room);
+
+                spannableString.setSpan(imageSpan_ACESS_TIME, 0, 1, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+                spannableString.setSpan(imageSpan_ROOM, 22, 23, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
```

Comment:
Hardcoding the span start and end won't work. I tested your branch with `ccc35c3` flavor. Here is an example screenshot:

![detailbar](https://user-images.githubusercontent.com/144518/68024454-717a6f80-fcaa-11e9-8a5b-bb70dea410b7.png)

Reply:
the template date and time consisted of a.m./p.m. at the end of the time. if the 12-hour format is not followed this issue is occurred. I did not test against 24-hour format. I could add an if-else statement to place the room icon based on the 12/24-hour format. But I don't know if the value from `dateUTC` specifies the 12/24-hour format.

Reply:
Please plan for this to work whatever date format might be chosen either by the system or in another language other then German or English.

Reply:
Now, I dropped the spannable string approach as the design is now done by layout instead of programmatic.